### DISCUSSION TITLE
cannot get dragonfly 3.2.8 working with Logic Pro, Mac Monterey 12.5.1

### DISCUSSION THREAD

**Discussion by keithfwalter** [michaelwillis/dragonfly-reverb#153] [General]
upon attempting to install on my mac, unrecognized developer error was received.  right clicking>Open With worked, but plugings do not show up in Logic Pro, though i can see them in the plugin folder.  I deleted all plugin files and tried again.  no joy.  rebooted.  no joy.  Anyone have any ideas how to get these working in Logic Pro?

**Comment by falkTX** [comment]
Logic only supports AU, right? these plugins do not have that format, for now

**Comment by keithfwalter** [comment]
well, that makes sense why I don't see them.  I thought Logic supported .vst's but nope, only .component (AU).  Thanks for the insight!
